multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -110,13 +139,16 @@ def infer_precision(
             )
             precision = 32
 
+        else:
```

Comment:
Our config precision `bf16` can still overwrite the precision in checkpoint?

Reply:
The config default is 16 not bf16.

Reply:
You are right. What if users specify `bf16` in config and the checkpoint precision is not `bf16`?

Reply:
User argument should be treated with the highest priority. I do not think it's a good idea to override them.